inference-engine/src/mkldnn_plugin/mkldnn_plugin.cpp

```diff
@@ -370,6 +371,7 @@ static void TransformationUpToCPUSpecificOpSet(std::shared_ptr<ngraph::Function>
     pass_config->disable<ngraph::pass::ConvertGather7ToGather1>();
     pass_config->disable<ngraph::pass::ConvertMinimum>();
     pass_config->disable<ngraph::pass::ConvertBroadcastToTiles>();
+    pass_config->disable<ngraph::pass::ConvertMaxPool8ToMaxPool1>();
```

Comment:
Why we are disabling it? Seems like it will prevent from creating stub outputs each time and as a consequence decrease memory consumption.